post_fastapi/api/comment/comment_endpoint.py

```diff
@@ -85,12 +84,13 @@ async def get_by_postid(post_id: int, page: int, service: CommentService = Depen
 
 
 @router.patch(
-    "",
+    "/{comment_id}",
     response_model=UpdateCommentResponse,
     dependencies=[Depends(verify_authority_dependency)],
     status_code=status.HTTP_200_OK,
 )
 async def update_comment(
+    comment_id:int,
```

Comment:
```suggestion
    comment_id: int,
```

이제 파이썬 PEP8 코드 스타일은 습관화 & 내제화 하세용